upstairs/src/volume.rs

```diff
@@ -473,6 +473,14 @@ impl BlockIO for Volume {
         BlockReqWaiter::immediate()
     }
 
+    fn write_unwritten(
+        &self,
+        _offset: Block,
+        _data: Bytes,
+    ) -> Result<BlockReqWaiter, CrucibleError> {
+        crucible_bail!(Unsupported, "write_unwritten not supported for volumes")
```

Comment:
Why not?

Reply:
I don't think we want to expose `write_unwritten` to the user.  It's really something that only exists for the scrubber and the scrubber should run "inside" the volume layer.